bus/bus.go

```diff
@@ -500,6 +503,9 @@ func (b *Bus) Handler() http.Handler {
 		"GET    /state":         b.stateHandlerGET,
 		"GET    /stats/objects": b.objectsStatshandlerGET,
 
+		"GET    /state/autopilot":        b.stateAutopilotHandlerGET,
```

Comment:
I'd drop the `state` here and group all 4 endpoints under `/autopilot` to have them in one place.